cli/README.md

```diff
@@ -81,3 +82,45 @@ Global Options:
       Path to kubeconfig file. This is aliased as "-c".
 
 ```
+
+### consul-k8s uninstall
+This command uninstalls Consul on Kubernetes, while prompting whether to uninstall the release and whether to delete all
+related resources such as PVCs, Secrets, and ServiceAccounts.
+
+Get started with:
+```bash
+consul-k8s uninstall
+```
+
+```
+Usage: kubectl consul uninstall [options]
+Uninstall Consul with options to delete data and resources associated with Consul installation.
+
+Command Options:
+
+  -auto-approve
+      Skip approval prompt for uninstalling Consul. The default is false.
+
+  -name=<string>
+      Name of the installation. This can be used to uninstall and/or delete
+      the resources of a specific Helm release.
+
+  -namespace=<string>
+      Namespace for the Consul installation.
+
+  -wipe-data
+      This behaviour of this flag depends on the value of -auto-approve.
+      When -wipe-data=true, the CLI will delete PVCs and Secrets if
+      -auto-approve=true. When -wipe-data=false, the CLI will prompt to
+      wipe PVCs and Secrets if -auto-approve=false, and skip wiping them
+      if -auto-approve=true. Only set this to true when persisted data from
+      previous installations is no longer necessary. The default is false.
```

Comment:
Note: the latest commit on this branch is the `-wipe-data` behaviour discussed in eng-sync. I realized when doing this that it's kind of confusing behaviour to explain in a flag description. 

The confusing part is that -wipe-data=false means 2 different things, based on what -auto-approve is.

What if we had `auto-approve` and `-auto-approve-wipe-data`, where the former would just uninstall with no prompts and the latter would uninstall and wipe data with no prompts. If both are false, the CLI will prompt both.

Reply:
Chatted with David and Luke at cli sync and Luke had a nice way to phrase this flag:

"When -wipe-data is set to true in combination with -auto-approve=true, uninstall will delete all persisted data (PVCs and Secrets) from previous installations."

Then we'd have flag validation for if you set it alone, it will error out. 

So I'll go ahead and update this!